frontend/src/api/utils.ts

```diff
@@ -21,6 +21,21 @@ export async function getCollateralTokenSymbols(loans: Loan[]): Promise<string[]
   return Promise.all(loans.map((l) => l.parameters.collateralToken.symbol()));
 }
 
+export async function getPossibleWithdrawals(loans: Loan[]): Promise<Withdrawal[][]> {
```

Comment:
Taking a step back, all of `frontend/src/api` shouldn't have to exist. My approach for now has been to let it live according to what's comfortable to consume by the frontend, hopefully teaching us a bit about usage patterns. Then we can slowly merge such patterns into `getline.ts` so that future users don't have to write such amount of boilerplate again.

See [Desire paths](https://en.wikipedia.org/wiki/Desire_path). :)